experimental/cloudstrapper/playbooks/roles/control/files/v1.3/orc8r-aws/eks.tf

```diff
@@ -0,0 +1,74 @@
+################################################################################
```

Comment:
I think this whole section of versioned `.tf` files is unnecessary. We just need to provide a templated main.tf for v1.3 and v1.4 Terraform allows setting a reference to a github repo which we set for example like this
```
module orc8r {
  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.4"
  # ...
}

module orc8r-app {
  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref=v1.4"
  # ...
}
```
this will use the tf files from the repo rather than those locally.

Reply:
I put in this file just to overcome some of the fixes that hadn't made it to the v1.3 release when I was building the tool. Once backported to 1.3, we can simply remove the call to patch-terraform.yaml altogether. For example, 1.4 does not have any patching requirement since the code works out of the box.

Since 1.3 would hang around only till 1.5, I will leave this as-is. Although a bit kludgy, it deploys 1.3 as expected and is used primarily for upgrade testing only at this time. 

WDYT?

Reply:
The fixes that need a backport to v1.3 are
1. orc8r-aws: Needs to specify cluster_version in eks.tf to 1.17 or higher.
2. orc8r-helm-aws: Needs to specify stable_helm_repo and incubator_helm_repo to charts.helm.sh